src/helpers/required.ts

```diff
@@ -62,7 +62,7 @@ export namespace required {
    * Show Requirements Page
    */
   export async function checkRequiredApps(): Promise<boolean> {
-    return checkApps(...requiredApps);
+    return await checkApps(...requiredApps);
```

Comment:
In this case, I've made the changes here, in this PR, and after that, I realized that it might be better convert those tweaks into a standalone PR. :)